src/cosmosdb-preview/setup.py

```diff
@@ -16,7 +16,7 @@
 
 # TODO: Confirm this is the right version number you want and it matches your
 # HISTORY.rst entry.
-VERSION = '1.0.1'
+VERSION = '1.0.1b2'
```

Comment:
Why `1.0.1b2`? There's no `1.0.1b1` before🤔

Reply:
@AllyW Can you help with the version definition?

Reply:
Hi @carjackson-msft cosmos-db-preview `1.0.1` is `preview` version cause it has tag ` "azext.isPreview": true`  here https://github.com/Azure/azure-cli-extensions/blob/f5f828991a4ec4cfe290cfeed3b25ddd9450480b/src/cosmosdb-preview/azext_cosmosdb_preview/azext_metadata.json#L2.

Therefore, after we have enforced new semantic versioning schema this May (https://github.com/Azure/azure-cli/blob/dev/doc/extensions/versioning_guidelines.md#versioning-of-azure-cli-extension), it takes the old release version as `1.0.1b1` and the next version would be `1.0.1b2`, according to this transition table: https://github.com/Azure/azure-cli/blob/dev/doc/extensions/versioning_guidelines.md#cli-extension-version-transition-table

Since this pr actually modified cmd interfaces, `1.1.0b1` makes sense, so `minor` label is added here to enable `1.1.0b1` for this pr. If you have other concerns, please let me know.

Reply:
@AllyW @evelyn-ys Update the minor versioning to 1.1.0b1 instead